src/helpers/print_helpers.rs

```diff
@@ -72,9 +72,23 @@ pub fn print_book(sdk: &SDKClient, book: &Ladder) {
 }
 
 pub fn get_precision(mut target: u64) -> usize {
+    let is_prime = |x| {
```

Comment:
this seems overly complicated. why not just divide by 2 until it's no longer divisible by 2, and divide by 5 until it's no longer divisible by 5?